etc/kernel-launchers/scala/toree-launcher/src/main/scala/launcher/ToreeLauncher.scala

```diff
@@ -239,7 +249,7 @@ object ToreeLauncher {
     if (sigNum == 9) "KILL"
     else {
       if (alternateSigint == null) {
-        println("WARNING: --alternate-sigint is not defined and signum %d has been " +
+        logger.info("WARNING: --alternate-sigint is not defined and signum %d has been " +
```

Comment:
Should probably use `logger.warn`.  We could also remove the `WARNING:` prefix as well.